Xamarin.Forms.Platform.MacOS/Renderers/EntryRenderer.cs

```diff
@@ -6,18 +6,6 @@ namespace Xamarin.Forms.Platform.MacOS
 {
 	public class EntryRenderer : ViewRenderer<Entry, NSTextField>
 	{
-		class BoolEventArgs : EventArgs
```

Comment:
Did this moved to some other place ?

Reply:
Yea stupid me, forgot to check in the added file.  Its up now.  I moved it because it is now used in three places TimePicker, DatePicker, & Entry.  The reality is, it will most likely need to be used in more places.  Check that out again and if you want to move the BoolEventArgs somewhere else, like a separate file then we can do that.